pkg/common/idutil/safety.go

```diff
@@ -176,18 +144,130 @@ func IDFromProto(id *types.SPIFFEID) (spiffeid.ID, error) {
 // FormatPath formats a path string. The function ensures a leading slash is
 // present.
 func FormatPath(format string, args ...interface{}) string {
+	// TODO: when the safety valve is removed, this function should:
+	// 1. not ensure the leading slash.
+	// 2. validate that the produced path is correct
```

Comment:
I am wondering about this ... shouldn't we do this with 1.0 by default, and only do `ensureLeadingSlash` if the safety valve is activated?

Reply:
We could. However, these two helpers are currently only used by the registrar, which doesn't have a way to enable the safety valve.